### PR TITLE
python3-ansible-lint: update to 6.16.1

### PR DESCRIPTION
**Pull request by pascal-huber**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

**Comment by pascal-huber** [comment]
Added update for `python3-ansible-compat`

**Comment by jcgruenhage** [comment]
Tested and works. `v6.16.2` is out though, and I've tested this with this diff as well and it also works as expected. Can you update it accordingly?

```diff

diff --git a/srcpkgs/python3-ansible-lint/template b/srcpkgs/python3-ansible-lint/template
index 26f473abb3..83ae6adc9a 100644
--- a/srcpkgs/python3-ansible-lint/template
+++ b/srcpkgs/python3-ansible-lint/template
@@ -1,6 +1,6 @@
 # Template file for 'python3-ansible-lint'
 pkgname=python3-ansible-lint
-version=6.16.1
+version=6.16.2
 revision=1
 build_style=python3-pep517
 make_install_target="dist/ansible_lint-*-*-*-*.whl"
@@ -18,7 +18,7 @@ maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
 license="GPL-3.0-only"
 homepage="https://github.com/ansible/ansible-lint"
 distfiles="${PYPI_SITE}/a/ansible-lint/ansible-lint-${version}.tar.gz"
-checksum=1f2b796379f86e2dc57f1bfe4288a46c976a1d406d6221ee46e3ac8307f73ca3
+checksum=93ad8a04adcda6025d4ff218a10694120fe70cb91da8ac9f85c5dc82b32456e4
 # cba anymore, the list of failing tests changes with every update
 make_check="no"
 ```

**Comment by pascal-huber** [comment]
> Tested and works. `v6.16.2` is out though, and I've tested this with this diff as well and it also works as expected. Can you update it accordingly?

Thanks! I updated the `python3-ansible-lint` template and also  the dependency `python3-ansible-compat` (4.1.0 war released today).

File: srcpkgs/python3-ansible-compat/template [hunk 1]
```diff
@@ -10,7 +10,8 @@ maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
 license="MIT"
 homepage="https://github.com/ansible/ansible-compat"
 distfiles="${PYPI_SITE}/a/ansible-compat/ansible-compat-${version}.tar.gz"
-checksum=d7dc5e4b7fade9b7375c568a24475b7be81024cac2a20caef3260ef0b51287b4
+checksum=bcc6b3cf4bb102e1b1b36a8639fee7ad18ddb1d99fb055d208c8a0f650a51621
+
```
Comment (classabbyamp) [inline srcpkgs/python3-ansible-compat/template:14]: spurious newline added

Comment (pascal-huber) [reply srcpkgs/python3-ansible-compat/template:14]: indeed, fixed.

**Pull request merged**